_data/docs.yml

```
@@ -123,6 +123,16 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  pylibcugraphops:
+    name: cuGraph-Ops
+    path: pylibcugraphops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
```

```suggestion
      legacy: 0
```

`cugraph-ops` only recently started publishing documentation, so they don't have legacy (`23.02`) docs